test/api.test.ts

```diff
@@ -0,0 +1,106 @@
+import assert from 'assert';
+import supertest from 'supertest';
+import app from '../src/app';
+import mongoose from 'mongoose';
+import userModel from '../src/models/userModel';
+
+beforeEach((done) => {
```

Comment:
move all database connection things a similar stuff to test specific util file. tests/util.ts and then import this file into all test suites